### PR TITLE
NUTCH-2038

### PR DESCRIPTION
**Pull request by asitang**
Made new changes suggested and corrections by Sebastian.

### CODE REVIEW

File: src/plugin/parsefilter-naivebayes/src/java/org/apache/nutch/parsefilter/naivebayes/NaiveBayesParseFilter.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  private String inputFilePath;
+  private String dictionaryFile;
+  private ArrayList<String> wordlist = new ArrayList<String>();
+
+  public boolean filterParse(String text) {
+
+    try {
+      return classify(text);
+    } catch (IOException e) {
+      // TODO Auto-generated catch block
```
Comment (chrismattmann) [inline src/plugin/parsefilter-naivebayes/src/java/org/apache/nutch/parsefilter/naivebayes/NaiveBayesParseFilter.java]: I asked for this to be removed.

File: src/plugin/parsefilter-naivebayes/src/java/org/apache/nutch/parsefilter/naivebayes/NaiveBayesParseFilter.java [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    } catch (IOException e) {
+      LOG.error(StringUtils.stringifyException(e));
+
+    }
+
+    try {
+
+      train();
+    } catch (Exception e) {
+      // TODO Auto-generated catch block
```
Comment (chrismattmann) [inline src/plugin/parsefilter-naivebayes/src/java/org/apache/nutch/parsefilter/naivebayes/NaiveBayesParseFilter.java]: are you seeing my comments? I asked for this to be removed.

Comment (asitang) [inline src/plugin/parsefilter-naivebayes/src/java/org/apache/nutch/parsefilter/naivebayes/NaiveBayesParseFilter.java]: Remove what exactly???

Comment (asitang) [inline src/plugin/parsefilter-naivebayes/src/java/org/apache/nutch/parsefilter/naivebayes/NaiveBayesParseFilter.java]: Well you mean the //TODO comments.... then alright

**Pull request closed without merging**